tests/unit-test/test_cache.c

```diff
@@ -51,6 +51,37 @@ void test_generate_uuid(void) {
   TEST_ASSERT_TRUE(test_uuid[0]);
 }
 
+void test_cache_list_push(void) {
+  char* key = TEST_UUID_LIST_NAME;
+  const char* value = TEST_UUID;
+
+  TEST_ASSERT_EQUAL_INT(SC_OK, cache_list_push(key, strlen(key), value, strlen(value), 0));
+}
+
+void test_cache_list_at(void) {
+  char* key = TEST_UUID_LIST_NAME;
+  char res[UUID_STR_LEN];
+
+  TEST_ASSERT_EQUAL_INT(SC_OK, cache_list_at(key, -1, res));
```

Comment:
Why the index is -1?

Reply:
It means the index of the last element in list.

Reply:
Do we use the same redis table(?) as the normal TA used in this test?

Reply:
Yes